shared/test/test_redis_table.rb

```diff
@@ -28,16 +28,15 @@ def test_redis_tables
     assert_raises(RedisTable::NotFound) { table.fetch(1) }
 
     value = {'name' => 'alice', 'age' => 7, 'male' => false}
-    inserted = table.insert(value)
+    row1 = table.insert(value)
 
     value_table2 = {'name' => 'bob', 'age' => 12, 'male' => true}
-    table2.insert(value_table2)
+    table2_row1 = table2.insert(value_table2)
 
     # Each table should only see its own inserts, not others in the same shard.
-    assert_equal row(value, 1), inserted
-    assert_equal [row(value, 1)], table.to_a
-    assert_equal row(value, 1), table.fetch(1)
-    assert_equal [row(value_table2, 1)], table2.to_a
+    assert_equal [row1], table.to_a
```

Comment:
the old `row(value, id)` model was also implicitly testing that the thing returned by `table.insert` preserves all the values in `value`. Are we explicitly testing that elsewhere? If not, I'd like to be.